src/seedu/addressbook/data/tag/Tagging.java

```diff
@@ -0,0 +1,36 @@
+package seedu.addressbook.data.tag;
+
+import java.util.ArrayList;
+
+/*
+ * Keeps track of tags being added or deleted
```

Comment:
A more precise description is `Represents an adding or removing a Tag to/from a Person`.